apps/easypid/src/features/didcomm/useDidCommCredentialNotificationSlides.tsx

```diff
@@ -19,12 +21,20 @@ export function useDidCommCredentialNotificationSlides({
   const toast = useToastController()
   const { acceptCredential, acceptStatus, declineCredential, credentialExchange, attributes, display } =
     useDidCommCredentialActions(credentialExchangeId)
+  const { activities } = useActivities({ filters: { entityId: credentialExchange?.connectionId } })
 
   const onCredentialAccept = async () => {
     await acceptCredential().catch(() => {
       toast.show('Something went wrong while storing the credential.', { customData: { preset: 'danger' } })
       onCancel()
     })
+    await addReceivedActivity(agent, {
+      entityId: credentialExchange?.connectionId as string,
+      name: display.issuer.name,
+      logo: display.issuer.logo,
+      backgroundColor: '#ffffff', // Default to a white background for now
+      credentialIds: [`w3c-credential-${credentialExchange?.id}`],
```

Comment:
the exchange record is different from the crednetial. If we want the actual credential id we should do  `credentialExchange .credentials[0].credentialRecordId`

Reply:
@janrtvld i think this was not resolved?

Reply:
@TimoGlastra hmm credentialExchange.credentials seems to be an empty array just after the credential is accepted. Could it be that it has not processed yet?